styles.css

```diff
@@ -1 +1,83 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+@media screen and (min-width: 320px) {
+
+    .container {
+        background-color: rgba(255, 239, 213, 0.94);
+        color: rgba(255, 255, 255, 0.91);
+        display: grid;
+        grid-template-columns: 1fr;
+        grid-template-rows: 50px auto 50px 50px 50px 50px;
+        grid-template-areas:
+        "header"
+        "main"
+        "aside1"
+    "aside2"
+    "aside3"
```

Comment:
probeer alles in 1 lijn te zetten voor de rest perfect!

Reply:
Je hebt gelijk!